scripts/train_lm.py

```diff
@@ -144,15 +144,70 @@ def WriteMetaparameters(metaparameters, ngram_order, num_train_sets, out_file):
             i += 1
     f.close()
 
-def FormatMetaparameters(metaparameters):
+def FormatMetaparameters(metaparameters, num_train_sets):
```

Comment:
We only load the `num_train_sets` variable when `args.bypass_metaparameter_optimization != None`. If you need to use this value, please move the loading part of code outside the if condition.